client/src/Components/BookCard/index.jsx

```diff
@@ -1,8 +1,30 @@
 import React from 'react';
-import { Card, Modal, Button, Image, Header } from 'semantic-ui-react';
+import { Card, Modal, Button, Image, Header, ModalActions } from 'semantic-ui-react';
+import axios from 'axios';
 
 const BookCard = ({ book }) => {
-  const { title, author, genre, year, imgURL } = book;
+  const { title, author, genre, year, imgURL} = book;
+
+  const handleDownload = () => {
+    axios({
+      method: 'get',
+      url: `/api/PDFs.files/${book.PDF}`,
+      responseType: 'blob',
+    }).then(response => {
+      const url = window.URL.createObjectURL(new Blob([response.data]));
+      const link = document.createElement('a');
+      link.href = url;
+      link.setAttribute('download', `${book.title}`);
+      document.body.appendChild(link);
+      link.click();
```

Comment:
Widzę że tu niezły hack leci. Rozważyłbym dodanie jakieś biblioteki która zajmie się pobraniem tego pliku.

Opcja minimum to dodać to rozszerzenie do pliku bo narazie te pdf sie jako txt zapisują
`` link.setAttribute('download', `${book.title}.pdf`);``